package.json

```diff
@@ -20,26 +20,21 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^4.3.0",
+    "@theforeman/vendor": "^4.7.1",
     "intl": "~1.2.5",
     "jed": "^1.1.1",
     "react-intl": "^2.8.0"
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@babel/preset-env": "7.9.5",
-    "@babel/plugin-transform-modules-systemjs": "7.9.0",
-    "@babel/plugin-transform-modules-commonjs": "7.9.0",
-    "@babel/plugin-transform-modules-amd": "7.9.0",
```

Comment:
Following our discussion on IRC, let's keep these pinning here as well to make sure npm doesn't try to be too clever (and also the `"babel-plugin-dynamic-import-node": "2.3.0"` below).

Reply:
Still needs addressing